docker/vep/Dockerfile.GRCh37

```diff
@@ -0,0 +1,59 @@
+FROM {{ hail_ubuntu_image.image }}
+
+ENV VEP_VERSION 85
+ENV DEBIAN_FRONTEND noninteractive
+ENV TZ Etc/UTC
+ENV VEP_DIR /vep
+
+RUN apt-get update && apt-get -y install \
+    apache2 \
+    build-essential \
+    cpanminus \
+    curl \
+    git \
+    libmysqlclient-dev \
+    libpng-dev \
+    libssl-dev \
+    locales \
+    manpages \
+    mysql-client \
+    openssl \
+    perl \
+    perl-base \
+    python3.8 \
```

Comment:
Why python3.8? This is gonna pull in a bunch of dependencies. hail_ubuntu already has python3.7.

Reply:
Not sure. I grabbed this code from you. I'll try taking it out and see if it works.